docs/remote-caching.md

```diff
@@ -39,18 +39,18 @@ The key benefit of this approach is that you get the upload for free if you anyw
 
 #### How to use inline caching
 
-In order to enable inline caching, simply add `--ci` in your invocation of `earthly` in your CI, or `--use-inline-cache` on individual developer's machines. If the `--push` command is also specified, the use of the cache will be read-write.
+In order to enable inline caching, add `--ci --use-inline-cache --save-inline-cache` in your invocation of `earthly` in your CI, or `--use-inline-cache` on individual developer's machines. If the `--push` command is also specified, the use of the cache will be read-write.
 
 In CI, read-only inline cache (typically in PR builds):
 
 ```bash
-earthly --ci +some-target
+earthly --ci --use-inline-cache --save-inline-cache +some-target
```

Comment:
should this be:

```suggestion
earthly --ci --use-inline-cache +some-target
```

since this is for "read-only inline cache (typically in PR builds)"